### ISSUE TITLE
Local profile suddenly shows faulty times

### ISSUE THREAD

**Issue by Loopi2712** [MilosKozak/AndroidAPS#2317]
Describe the bug
I use AAPS version 2.5.1 on a Samsung S7 and the latest Xdip version. File AAPS preferences on the device is upto date. I can import the correct parameters including the local profile.

When I toggle to "overview" and back to "local profil" the times in the profile are changed. The annex shows screen shots of the correct and changed (wrong) local profile.

The profile in the pump (Dana RS) remains ok.

![Screenshot_20191216-203945_AndroidAPS](https://user-images.githubusercontent.com/11147787/71549875-53d53800-29c5-11ea-9e1f-12d2b9f1867f.jpg)
![Screenshot_20191216-204024_AndroidAPS](https://user-images.githubusercontent.com/11147787/71549882-69e2f880-29c5-11ea-8af9-8eecb50e5f7c.jpg)

    Import prefferences and restart AAPS
    Check local profil -> ok
    Scroll down to 'overview'
    Scroll back -> wrong local profil

Type of CGM -> G6, Xdrip 2.18.2.88
   
Thanks for helping. Otherwise the AAPS runs perfect !!

Kirstin & Axel

**Comment by Chris2001** [comment]
No shit, but I had the same problem tonight, suddely the profile jumping from 05:00 AM to 01:AM. Only realised this in the morning after getting to low few hours after midnight.
:zipper_mouth_face:

**Comment by dv1** [comment]
I also encountered this issue. Mistakenly commented on it [in the Nightscout repo](https://github.com/nightscout/cgm-remote-monitor/issues/5295#issuecomment-569483012). So, I copy-paste my text here:

I can confirm this. Exact same problem here. Nightscout 13.0.1, Dexcom G6, xDrip 6d4d283-2019.10.27, AndroidAPS 2.5.1. One ugly workaround I found was to create ISF and IC entries for all 24 hours and fill them all. That way, the hours cannot be corrupted by this bug.
